### ISSUE TITLE
Is there anyone that is still botting/used to that hasn't been banned yet

### ISSUE THREAD

**Issue by Enviouse** [finndev/PokeBuddy#1247]
Trying to figure out if I bot safe eg close to me n leave it there will I eventually get banned

**Comment by mpdt1234** [comment]
Hackemon has been working for me for two days now. 
Started a new account and I'm level 15.

**Comment by popdukes** [comment]
i would love to try hackemon but it seems to complicated to install.
i dl java and i cant seem to get it to work

**Comment by mpdt1234** [comment]
https://www.youtube.com/watch?v=dULUWb8pWOI

**Comment by FlameSTeeK** [comment]
I used 2 different bots (The first one was too fast), for like 8 or 9 hours in the last 2 weeks and got like 3 soft bans of 2 hours for teletransporting and I haven't been permabaned yet, and there is no warning on my mail either.
Maybe is because I'm from the third world? Just lucky? I'm in Chile btw

**Comment by Enviouse** [comment]
U will be banned eventually there not banning everyone at once eg my mate got banned a week befor me n my other mate got banned a week after me

**Comment by lukebiggz21** [comment]
Yeah one of my accounts got banned as I tried to log in and the other as I was using poke buddy today :(

**Comment by gerthantaclops** [comment]
if your settings are right and you dont get greedy you will be fine i went from lvl 2 to lvl 26 in 4 days this is how i did it start a game on your phone/device pick your nickname, pokemon and gym faction so your account looks legit as possible level up 1-2 times and then shut down PG go into settings on your phone and copy down make model man, and all the info it asks you on hackemons device settings take out your phones battery and find your exact location on a gps website if you do not use your phone info and random generate you will get a notification from google of unrecognizable device has logged into your google guess what your now on the watchlist. so with your phone info and gps set got to speed put it on road walk/ fast human 8km per hr and set your pokemon catch fail rate to 10% and make sure you put ballspin on and set your delays and dont bot for more than 5hrs at a time its all about looking as human as possible and now with the update against bots you need to master this i have had 2 accounts ban on other bots but this one is the best if set right you wont trigger the speed sensor making you verify your a passenger or the i am a human sensor my best advice is to make 1 or 2 PTC accounts not on your main device you want to play on try to get the hang of it and once you do logon your main account and be even safer with that one my main account is lvl 32 and hasnt been banned last i used the bot on it a month ago
